multimodal/src/autogluon/multimodal/matcher.py

```diff
@@ -434,7 +438,7 @@ def fit(
         if self._validation_metric_name is None or self._eval_metric_name is None:
             validation_metric_name, eval_metric_name = infer_metrics(
                 problem_type=problem_type,
-                pipeline=self._pipeline,
+                is_matching=is_matching(self._pipeline),
```

Comment:
We can consider to use a function name other than `is_matching`. Because `is_matching` is used as an argument.

Reply:
Changed.